packages/site/pages/install.js

```diff
@@ -25,7 +25,13 @@ Use of the Core or Components requires the existence of the Design System normal
 npm install @pluralsight/ps-design-system-normalize
 \`\`\`
 
-Include this vanilla CSS in your application in a method appropriate for your project. For usage options, follow the [Core Usage docs](/core/usage) patterns.
+## Theming
+
+Most components support multiple themes and have a dependency on theme package. Install with:
```

Comment:
I would modify this.  We've tried hard to make sure that *all* components support theming.  Whether there's an actual diff between themes is less important.  It's the perception problem that I don't want to reawaken.

Maybe:

```
All components support theming and have a peer dependency on the Theme package. Install with:
```

I would also *move* this down to the "Use Components" section.